cmd/ctr/commands/commands.go

```diff
@@ -153,7 +153,7 @@ var (
 		},
 		cli.StringSliceFlag{
 			Name:  "device",
-			Usage: "add a device to a container",
+			Usage: "add a device or directory with devices to a container",
```

Comment:
suggest:
`"file path to a device to add to the container; or a directory tree of devices to add to the container"`

Reply:
Updated.